assets/css/admin.scss

```diff
@@ -3156,6 +3156,7 @@ img.help_tip {
 			border: 1px solid #ddd;
 			/* rtl:ignore */
 			border-right: 0;
+			margin-left: -3px;
```

Comment:
Only for the line directly after it. Good catch. The color box looks a tiny bit smaller in RTL mode because it has `margin-right: -3px`. 

Easy fix is to put `/* rtl:ignore */` directly above `margin-left: -3px;` http://rtlcss.com/learn/usage-guide/control-directives/